build/web-features.ts

```diff
@@ -1,16 +1,37 @@
 import { features } from "web-features";
+import { computeBaseline } from "compute-baseline";
 
-export function getWebFeatureStatus(...bcdKeys: string[]) {
-  if (bcdKeys.length === 0) {
-    return;
+import type { Doc } from "../libs/types/document.js";
+
+export function addBaseline(doc: Partial<Doc>) {
+  if (doc.browserCompat && !doc.mdn_url?.includes("/docs/MDN/")) {
+    if (doc.browserCompat.length !== 1) {
+      return;
+    }
+
+    const key = doc.browserCompat[0];
+    const { featureStatus, keyStatus } = getStatuses(key);
+
+    if (!featureStatus) {
+      return;
+    }
+
+    if (featureStatus.baseline !== keyStatus.baseline) {
+      return;
+    }
+
+    return featureStatus;
   }
+}
 
+function getStatuses(key: string) {
```

Comment:
Nit: Maybe it would help to call this `bcdKey` or `featureKey` instead to be explicit about what kind of key this is. Maybe even add JSDoc with an example `// e.g. "web.api.clipboard"` (?).